### DISCUSSION TITLE
Prefetched for DOMcontentloaded

### DISCUSSION THREAD

**Discussion by FernandoRub13** [livewire/livewire#4422] [3. Feature ideas]
It would be pretty cool to have a way to prefetch action after the DOM content was loaded, not only for mouseover.
Something like that:
```
<x-jet-secondary-button wire:click.prefetch="$toggle('open')" wire:loading.attr="disabled">
         Cancel // The action of cancel will be prefetched after all the content inside the DOM were loaded in the screen, not only for mouseover action.
</x-jet-secondary-button>
```

**Comment by joshhanley** [comment]
@FernandoRub13 I'm not sure what you mean. Can you explain in more detail how that would work and what would happen?

Thanks!
